src/dxtbx/model/beam.h

```diff
@@ -133,7 +133,7 @@ namespace dxtbx { namespace model {
     }
 
     /**
-     * @param direction The beam direction vector from sample to source
+     * @param direction The beam direction vector from source to sample
```

Comment:
Isn't the original correct? I note above (line 132) has `direction_ = -s0.normalize();`, suggesting that the direction variable goes opposite to the direction of the wavevector, therefore from sample back towards source

Reply:
I can't remember my reason for thinking this at the time...I'm convinced some format classes use `unit_s0` as direction when creating beam objects, but I'm gonna put this down to newborn sleep deprivation.